packages/components/managed-data-table/src/data-table-settings/data-table-settings-panel.styles.js

```diff
@@ -0,0 +1,28 @@
+import styled from '@emotion/styled';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+import { CloseIcon } from '@commercetools-uikit/icons';
+
+const Container = styled.div`
+  background-color: ${vars.colorNeutral95};
+  border-radius: ${vars.borderRadius6};
+  padding: ${vars.spacingM};
+`;
+
+const HeaderContainer = styled.div`
+  display: flex;
+  flex: 1;
+  justify-content: space-between;
+`;
+
+const CloseButton = styled(CloseIcon)`
```

Comment:
We should probably wrap the Icon with a `button` element to make it accessible. You can use the [AccessibleButton](https://github.com/commercetools/ui-kit/tree/master/packages/components/buttons/accessible-button) for this :)